src/Pact/Analyze/Eval/Term.hs

```diff
@@ -74,6 +74,13 @@ instance Analyzer Analyze where
 addConstraint :: S Bool -> Analyze ()
 addConstraint b = modify' $ latticeState.lasConstraints %~ (.&& b)
 
+genUninterpreted :: HasKind a => Analyze (S a)
+genUninterpreted = do
+  nextId <- use $ globalState.gasNextUninterpId
```

Comment:
`nextId <- globalState.gasNextUninterpId <<+= 1`

I think this would work if you want. I rarely have the opportunity to use the passthrough operators.

Reply:
Interesting! Didn't know about that one. But given stu's recent comment on "operator soup" we should probably stay conservative here